app/views/gardens/show.html.erb

```diff
@@ -0,0 +1,43 @@
+<center>
+  <article class='left-col'>
+    <section class='garden-info'>
+      <h1><%= @garden.name %></h1>
```

Comment:
@leahriffell add location and description; static hardcode an image

Reply:
@dcoleman21 done!